src/libs/actions/User.js

```diff
@@ -284,6 +284,33 @@ function subscribeToUserEvents() {
         });
 }
 
+/**
+ * Subscribes to Expensify Card updates when checking loginList for private domains
+ */
+function subscribeToExpensifyCardUpdates() {
+    const pusherChannelName = `private-user-accountID-${currentUserAccountID}`;
+
+    // Handle Expensify Card approval flow updates
+    Pusher.subscribe(pusherChannelName, Pusher.TYPE.EXPENSIFY_CARD_UPDATE, (pushJSON) => {
+        if (pushJSON.isUsingExpensifyCard) {
+            Onyx.merge(ONYXKEYS.USER, {isUsingExpensifyCard: pushJSON.isUsingExpensifyCard, isCheckingDomain: null});
```

Comment:
Setting it to `null` should delete the key from Onyx. Since this is a one-time only event, I felt like it was better to remove the key than leave it set as `false`.